package.json

```diff
@@ -220,7 +220,7 @@
         "react-modal": "^3.5.1",
         "react-process-string": "^1.2.0",
         "react-styleguidist": "^8.0.6",
-        "react-tether": "^0.6.1",
+        "react-tether": "^1.0.4",
```

Comment:
use fixed version - `1.0.3` because `1.0.4` cause uglifyJS to crash during a webpack prod run